attestation-service/verifier/Cargo.toml

```diff
@@ -4,22 +4,34 @@ version = "0.1.0"
 edition = "2021"
 
 [features]
-default = [ "all-verifier" ]
-all-verifier = [ "tdx-verifier", "sgx-verifier", "snp-verifier", "az-snp-vtpm-verifier", "az-tdx-vtpm-verifier", "csv-verifier", "cca-verifier" ]
-tdx-verifier = [ "eventlog-rs", "scroll", "sgx-dcap-quoteverify-rs" ]
-sgx-verifier = [ "scroll", "sgx-dcap-quoteverify-rs" ]
-az-snp-vtpm-verifier = [ "az-snp-vtpm", "sev", "snp-verifier" ]
-az-tdx-vtpm-verifier = [ "az-tdx-vtpm", "openssl", "tdx-verifier" ]
-snp-verifier = [ "asn1-rs", "openssl", "sev", "x509-parser" ]
-csv-verifier = [ "openssl", "csv-rs", "codicon" ]
-cca-verifier = [ "ear", "veraison-apiclient" ]
+default = ["all-verifier"]
```

Comment:
nit: we should avoid formatting the Cargo.toml file. edit w/ nano or something :)

Reply:
Apart from the fact that this is unrelated to the PR, is there a reason to not format `Cargo.toml` file? I see that the [documentation](https://github.com/rust-lang/rust/blob/master/src/doc/style-guide/src/cargo.md) does recommend breaking untenably long lines into multi-line block.

Reply:
Formatting toml is fine per se, IMO. It's problematic when it's not enforced in the CI and different editors disagree on how to format. In this case there will be bigger diffs flapping between different styles and it's hard to find meaningful changes when you have to bisect.